java/server/src/org/openqa/grid/internal/utils/SelfRegisteringRemote.java

```diff
@@ -181,7 +181,7 @@ public void startRegistrationProcess() {
     }
 
     if (!register) {
-      LOG.info("No registration sent ( register = false )");
+      LOG.fine("No registration sent ( register = false )");
```

Comment:
It was transformed to `finest` in the previous PR: https://github.com/SeleniumHQ/selenium/pull/7170/files#r281601307

The author says: "The log level here is too low. It should be `info`"